samples/cellular/udp/Kconfig

```diff
@@ -14,6 +14,13 @@ config UDP_DATA_UPLOAD_FREQUENCY_SECONDS
 	int "How often data is transmitted to the server"
 	default 900
 
+config UDP_DATA_UPLOAD_ITERATIONS
```

Comment:
Do we want a range here as well, as in the download client?

Reply:
I thought about it and to me it doesn't make sense to add it